CMakeLists.txt

```diff
@@ -79,6 +79,8 @@ elseif("${TARGET_ARCH}" STREQUAL "i686")
   iotjs_add_compile_flags(-D__i686__ -D__x86__ -march=i686 -m32)
 elseif("${TARGET_ARCH}" STREQUAL "x86_64")
   iotjs_add_compile_flags(-D__x86_64__)
+elseif("${TARGET_ARCH}" STREQUAL "gbs")
+  iotjs_add_compile_flags(-D__gbs__)
```

Comment:
@haesik Could you explain why `__gbs__` is defined here? I cannot find __gbs__ enclosed code in this PR. Furthermore identifier with leading double underscores is reserved by standard.

Reply:
I Just add for the future use..  Ok, I will remove it now..